frontend/src/store/modules/gotEnzymes.js

```diff
@@ -9,33 +9,51 @@ const data = {
 };
 
 const actions = {
-  async getReactionData({ commit }, id) {
-    const { info, crossReferences } = await gotEnzymesApi.fetchReaction(id);
-
+  async getCompoundData({ commit }, id) {
+    const { info, crossReferences } = await gotEnzymesApi.fetchCompound(id);
     commit('setInfo', info);
     commit('setCrossReferences', crossReferences);
   },
-  async getCompoundData({ commit }, id) {
-    const { info, crossReferences } = await gotEnzymesApi.fetchCompound(id);
 
+  async getDomainData({ commit }, id) {
+    const { info } = await gotEnzymesApi.fetchDomain(id);
     commit('setInfo', info);
-    commit('setCrossReferences', crossReferences);
```

Comment:
This is needed or else the the domain page can incorrectly display cross references from another component. For example: `/gotenzymes` -> Click the link for `R01677` -> Click back -> Click the link for `eukaryotes`. Similarly this needs to be added for `getOrganismData`. This is not the most DRY solution so happy for other suggestions too.

```suggestion
    commit('setCrossReferences', []);
```

Reply:
Instead of doing this in the store, I'm more in favour of returning `[]` from the API, making the API response more consistent.

Reply:
Addressed in: https://github.com/MetabolicAtlas/MetabolicAtlas/pull/1062/commits/42ce1b74bae43c2253e4497d9c7fea20ea136089